### ISSUE TITLE
the results of Demuxlet dataset

### ISSUE THREAD

**Issue by JuanruMaryGuo** [EDePasquale/DoubletDecon#37]
Hi, I'm trying to run DoubletDecon using Demuxlet dataset by running 20 times. However, I haven't gotten a result with high accuracy, I'm wondering if there are some errors in my code. Would you please share your code for reference?

By the way, I failed to run the code with the rho bigger than or equal to 1 and finally made it to be 0.61 after consulting other issues in Github. Might it cause the problem?

Thank you in advance!

**Comment by EDePasquale** [comment]
Hi,

Are you doing each run with the exact same parameters?

**Comment by JuanruMaryGuo** [comment]
Yes, I just ran the same code 20 times and all of the parameters are the same in the example except rho = 0.61.
